public/app/features/dashboard-scene/settings/DashboardEditView.tsx

```diff
@@ -0,0 +1,60 @@
+import { useLocation } from 'react-router-dom';
+
+import { locationUtil, NavModelItem } from '@grafana/data';
+import { SceneObject } from '@grafana/scenes';
+import { t } from 'app/core/internationalization';
+import { getNavModel } from 'app/core/selectors/navModel';
+import { useSelector } from 'app/types';
+
+import { DashboardScene } from '../scene/DashboardScene';
+
+import { AnnotationsEditView } from './AnnotationsEditView';
+import { GeneralSettings } from './GeneralSettings';
+import { VariablesEditView } from './VariablesEditView';
+
+export interface DashboardEditView extends SceneObject {
+  getUrlKey(): string;
+}
+
+export function useDashboardEditPageNav(dashboard: DashboardScene, currentEditView: string) {
+  const location = useLocation();
+  const navIndex = useSelector((state) => state.navIndex);
+  const navModel = getNavModel(navIndex, 'dashboards/browse');
```

Comment:
I don't fully understand why this `dashboards/browse` ID. Probably my lack of NavModel context. I'll investigate further 😄

Reply:
@ivanortegaalba it's a from a very old redux store that all pages use to get the correct page title & breadcrumbs